Makefile

```diff
@@ -0,0 +1,24 @@
+SHELL=/bin/bash
```

Comment:
What does this file do? wouldn't it suffice to call make in the scores directory?